Feature/storybook (#40)
infinum/eightshift-frontend-libs: blocks/init/components/carousel-navigation/docs/readme.md

# Carousel Navigation Component

> Component used primarly in carousel navigation block to provide navigation for switching slides left/right.

Reviewer: ```suggestion
A component used primarily in the carousel navigation block to provide navigation for switching slides left/right.
```

## Dependencies

None

> ## Implementation

Reviewer 2: ```suggestion
## Implementation

```

1. Copy/Paste component folder in your project
2. Rename `eightshift-boilerplate` namespace to your project
3. Implement project specific styles.

> 4. Implement in your block/s by providing necesery attributes.

Reviewer: ```suggestion
4. Implement in your block/s by providing necessary attributes.
```

> 4. Implement in your block/s by providing necesery attributes.

Reviewer 2: ```suggestion
4. Implement in your block(s) by providing necessary attributes.
```
